### ISSUE TITLE
Make the selection/customization of calendar colors a school configurable option

### ISSUE THREAD

**Issue by saschaben** [ilios/frontend#2910]
Schools should be allowed to select and manage their own palette of session type colors. This will both provide the ability to address legacy issues without changing code, and will also allow for the (slow) migration away from the use of specific colors to deliver content information, and toward a more accessible and ADA compliant approach to delivery.
![colorpicker](https://cloud.githubusercontent.com/assets/1753431/25411739/42627852-29d3-11e7-851b-c138d9b15b0a.png)
![colorpicker2](https://cloud.githubusercontent.com/assets/1753431/25411738/42625c28-29d3-11e7-957d-418a4e2da074.png)
![colorpicker3](https://cloud.githubusercontent.com/assets/1753431/25411740/42635290-29d3-11e7-88d2-8c72717e9d0c.png)

**Comment by saschaben** [comment]
A few things that are *not* in the mocks above, that we should discuss: 
1) considering the number of session types that will be in the list, do we want to provide a filter?
2) if there is a quick'n'dirty color picker that someone has for ember, do we want to just plug it in?
3) is there anything I've forgotten?

**Comment by jrjohnson** [comment]
There is very little extra work to add full blown session type editing. Create new, edit title, delete, etc... Is there any reason to restrict this to just colors and not all session type info?

**Comment by saschaben** [comment]
trans "Calendar Session Colors": 
FR: `Couleurs des Séance de Calendrier`
SP: `Colores de Sesiones de Calendario`

**Comment by saschaben** [comment]
trans "Instructional Method": 
FR: `modalités d'instruction`
SP: `métodos de instrucción`

trans "Assessment Method":
FR: `méthodes d'évaluation`
SP: `métodos de evaluación`
